### PR TITLE
Translate to Brazilian Portuguese the config section of the basic training

### PR DESCRIPTION
**Pull request by iaradsouza1**
# Description

Added first version for config.

Review translation and add example for conda config.

### CODE REVIEW

File: docs/basic_training/config.pt.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+name = value
+```
+
+```groovy linenums="1"
+process.conda = "/home/ubuntu/miniconda2/envs/nf-tutorial"
+```
+
+!!! info
+
+Observe que *strings* precisam ser colocadas entre aspas, enquanto números e valores booleanos (`true`, `false`) não. Além disso, observe que os valores são tipados, o que significa, por exemplo, que `1` é diferente de `'1'`, pois o primeiro é interpretado como o número um, enquanto o último é interpretado como uma *string*.
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:34]:
```suggestion
     Observe que *strings* precisam ser colocadas entre aspas, enquanto números e valores booleanos (`true`, `false`) não. Além disso, observe que os valores são tipados, o que significa, por exemplo, que `1` é diferente de `'1'`, pois o primeiro é interpretado como o número um, enquanto o último é interpretado como uma *string*.
```

File: docs/basic_training/config.pt.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
+---
+title: Configuration
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:2]:
```suggestion
title: Configuração
```

File: docs/basic_training/config.pt.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+description: Basic Nextflow Training Workshop
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:3]:
```suggestion
description: Material de treinamento básico do Nextflow
```

File: docs/basic_training/config.pt.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+Quando existir mais de um dos arquivos acima, eles serão mesclados, de modo que as configurações do primeiro substituam as mesmas configurações que podem aparecer no segundo e assim por diante.
+
+O mecanismo de pesquisa padrão do arquivo de configuração pode ser estendido fornecendo um arquivo de configuração extra usando a opção de linha de comando: `-c <arquivo de configuração>`.
+
+### Sintaxe do arquivo de configuração
+
+Um arquivo de configuração do Nextflow é um arquivo de texto simples contendo um conjunto de propriedades definidas usando a sintaxe:
+
+```groovy linenums="1"
+name = value
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:25]:
```suggestion
nome = valor
```

File: docs/basic_training/config.pt.md [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+!!! info
+
+Observe que *strings* precisam ser colocadas entre aspas, enquanto números e valores booleanos (`true`, `false`) não. Além disso, observe que os valores são tipados, o que significa, por exemplo, que `1` é diferente de `'1'`, pois o primeiro é interpretado como o número um, enquanto o último é interpretado como uma *string*.
+
+### Variáveis de configuração
+
+As propriedades de configuração podem ser usadas como variáveis no próprio arquivo de configuração, usando a sintaxe usual `$propertyName` ou `${expression}`.
+
+```groovy linenums="1"
+propertyOne = 'world'
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:41]:
```suggestion
propriedadeUm = 'mundo'
```

File: docs/basic_training/config.pt.md [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+anotherProp = "Hello $propertyOne"
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:42]:
```suggestion
umaOutraPropriedade = "Olá $propriedadeUm"
```

File: docs/basic_training/config.pt.md [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+customPath = "$PATH:/my/app/folder"
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:43]:
```suggestion
caminhoCustomizado = "$PATH:/pasta/da/minha/app"
```

File: docs/basic_training/config.pt.md [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+  */
+```
+
+### Configurar escopo
+
+As definições de configuração podem ser organizadas em diferentes escopos. Pode-se utilizar a notação `escopo.propriedade` ou agrupar as propriedades no mesmo escopo usando a notação de colchetes, como mostrado a seguir:
+
+```groovy linenums="1"
+alfa.x = 1
+alpha.y = 'valor da string..'
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:69]:
```suggestion
alfa.y = 'valor da string..'
```

File: docs/basic_training/config.pt.md [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+         ```
+
+     Compare o resultado das duas execuções.
+
+### Configurar ambiente
+
+O escopo `env` permite a definição de uma ou mais variáveis que serão exportadas para o ambiente onde serão executadas as tarefas do *workflow*.
+
+```groovy linenums="1"
+env.ALPHA = 'algum valor'
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:130]:
```suggestion
env.ALFA = 'algum valor'
```

File: docs/basic_training/config.pt.md [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+env.BETA = "$HOME/some/path"
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:131]:
```suggestion
env.BETA = "$HOME/algum/caminho"
```

File: docs/basic_training/config.pt.md [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+Por fim, execute o seguinte comando:
+
+```bash
+nextflow execute foo.nf -c my-env.config
+```
+
+??? result
+
+     ```console
+    BETA=/home/some/path
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:154]:
```suggestion
    BETA=/home/algum/caminho
```

File: docs/basic_training/config.pt.md [hunk 12]
```diff
@@ -0,0 +1,303 @@
...
+    ALPHA=algum valor
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:155]:
```suggestion
    ALFA=algum valor
```

File: docs/basic_training/config.pt.md [hunk 13]
```diff
@@ -0,0 +1,303 @@
...
+Isso é útil ao criarmos um exemplo-teste ou um protótipo de fluxo de trabalho.
+
+No entanto, é sempre uma boa prática desacoplar a lógica de execução do fluxo de trabalho das configurações do processo. Assim, é altamente recomendável definir as configurações do processo no arquivo de configuração do fluxo de trabalho em vez de no script do fluxo de trabalho.
+
+O escopo de configuração do `process` permite a configuração de quaisquer [diretivas] do `process` (https://www.nextflow.io/docs/latest/process.html#directives) no arquivo de configuração do Nextflow. Por exemplo:
+
+```groovy linenums="1"
+process {
+     CPU = 10
+     memória = 8.GB
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:171]:
```suggestion
     memory = 8.GB
```

File: docs/basic_training/config.pt.md [hunk 14]
```diff
@@ -0,0 +1,303 @@
...
+
+Isso é útil ao criarmos um exemplo-teste ou um protótipo de fluxo de trabalho.
+
+No entanto, é sempre uma boa prática desacoplar a lógica de execução do fluxo de trabalho das configurações do processo. Assim, é altamente recomendável definir as configurações do processo no arquivo de configuração do fluxo de trabalho em vez de no script do fluxo de trabalho.
+
+O escopo de configuração do `process` permite a configuração de quaisquer [diretivas] do `process` (https://www.nextflow.io/docs/latest/process.html#directives) no arquivo de configuração do Nextflow. Por exemplo:
+
+```groovy linenums="1"
+process {
+     CPU = 10
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:170]:
```suggestion
     cpus = 10
```

File: docs/basic_training/config.pt.md [hunk 15]
```diff
@@ -0,0 +1,303 @@
...
+    ```groovy linenums="1"
+    process foo {
+    cpus 4
+    memory 2.GB
+    time 1.hour
+    maxRetries 3
+
+    script:
+    """
+        your_command --cpus $task.cpus --mem $task.memory
```
Comment (mribeirodantas) [inline docs/basic_training/config.pt.md:204]:
```suggestion
        seu_comando --cpus $task.cpus --mem $task.memory
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
I'm not sure if you're finished with this translation, as it's still a draft, but I already sent some suggestions for you to approve if you agree.

File: docs/basic_training/config.pt.md [hunk 16]
```diff
@@ -157,13 +156,13 @@ nextflow execute foo.nf -c my-env.config
 
 ### Configurar processos
 
-`process` [directives](https://www.nextflow.io/docs/latest/process.html#directives) permitem a especificação de configurações para a execução da tarefa, como `cpus`, `memory`, `container`, e outros recursos no script do pipeline.
+A [diretiva](https://www.nextflow.io/docs/latest/process.html#directives) `process`  permite a especificação de configurações para a execução de uma tarefa, como `cpus`, `memory`, `container`, além de outros recursos, no script do pipeline.
 
-Isso é útil ao criarmos um exemplo-teste ou um protótipo de fluxo de trabalho.
+Isso é útil ao criarmos um exemplo-teste ou um protótipo para o nosso pipeline.
```
Comment (iaradsouza1) [inline docs/basic_training/config.pt.md:161]: I simplified the term "workflow" to "pipeline", as I think "workflow" here means the pipeline as a whole

File: docs/basic_training/config.pt.md [hunk 17]
```diff
@@ -286,18 +285,18 @@ Os seguintes protocolos são suportados:
...
-     !!! observação
+    !!! note
 
-         O Nextflow irá baixar a imagem do contêiner automaticamente. Isso levará alguns segundos, dependendo da velocidade da conexão de rede.
+        O Nextflow irá baixar a imagem do contêiner automaticamente. Isto levará alguns segundos, dependendo da velocidade da sua conexão. 
 
 ### Configurar execução com ambientes Conda
 
-O uso de um ambiente Conda também pode ser fornecido no arquivo de configuração adicionando a seguinte configuração no arquivo `nextflow.config`:
+Ambientes Conda também podem ser fornecidos no arquivo de configuração. Basta adicionar a seguinte configuração no arquivo `nextflow.config`:
```
Comment (iaradsouza1) [inline docs/basic_training/config.pt.md:301]: I think some example code is missing here

File: docs/basic_training/config.pt.md (hunk 16 above)
Comment (mribeirodantas) [reply docs/basic_training/config.pt.md:161]: 🥳

**Review by iaradsouza1 — COMMENTED** [review]
I reviewed the text, and mostly changed some phrases to improve understanding. Also, I fixed some toggle sections.

File: docs/basic_training/config.pt.md (hunk 17 above)
Comment (mribeirodantas) [reply docs/basic_training/config.pt.md:301]: Thanks for pointing this out, Iara! For some reason, this is missing in the new layout (but still there in training.seqera.io). The line missing is: process.conda = "/home/ubuntu/miniconda2/envs/nf-tutorial". You can add it here. I will add it to the original version in English now.

Comment (mribeirodantas) [reply docs/basic_training/config.pt.md:301]: Fixed in https://github.com/nextflow-io/training/pull/189

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**